Source/WebCore/platform/graphics/BitmapImageSource.cpp

```diff
@@ -743,6 +743,13 @@ RepetitionCount BitmapImageSource::repetitionCount() const
     return imageMetadata(m_repetitionCount, static_cast<RepetitionCount>(RepetitionCountNone), CachedFlag::RepetitionCount, &ImageDecoder::repetitionCount);
 }
 
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+bool BitmapImageSource::shouldUseQuickLookForFullscreen() const
+{
+    return m_decoder->shouldUseQuickLookForFullscreen();
+}
+#endif
```

Comment:
Do we need to cache the return of this metadata? If yes, you can follow the pattern of the uti for example.